src/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -212,14 +238,41 @@ export class DashboardListing extends React.Component {
       return (
         <React.Fragment>
           <EuiCallOut
-            title="Listing limit exceeded"
+            title={
+              <FormattedMessage
+                id="kbn.dashboard.listing.listingLimitExceeded.title"
+                defaultMessage="Listing limit exceeded"
+              />
+            }
             color="warning"
             iconType="help"
           >
             <p>
-              You have {this.state.totalDashboards} dashboards,
-              but your <strong>listingLimit</strong> setting prevents the table below from displaying more than {this.props.listingLimit}.
-              You can change this setting under <EuiLink href="#/management/kibana/settings">Advanced Settings</EuiLink>.
+              <FormattedMessage
+                id="kbn.dashboard.listing.listingLimitExceeded.description"
+                // eslint-disable-next-line max-len
+                defaultMessage="You have {totalDashboards} dashboards, but your {listingLimitText} setting prevents the table below from displaying more than {listingLimitValue}. You can change this setting under {advancedSettingsLink}."
+                values={{
+                  totalDashboards: this.state.totalDashboards,
+                  listingLimitValue: this.props.listingLimit,
+                  listingLimitText: (
+                    <strong>
+                      <FormattedMessage
+                        id="kbn.dashboard.listing.listingLimitExceeded.listingLimitText"
+                        defaultMessage="listingLimit"
```

Comment:
I think we don't need to translate this text because it's a kind of property
From Advanced Settings:
![image](https://user-images.githubusercontent.com/11524637/47902518-4391fd80-de93-11e8-9d57-8ae943ac392e.png)